string camera_name
string video_device

string unload
---